sp800-63b/sec5_authenticators.md

```diff
@@ -359,7 +359,7 @@ Biometric matching SHOULD be performed locally on claimant's device or MAY be pe
 
 If matching is performed centrally:
 
-* Use of the biometric SHALL be bound tightly to a single, specific device that is identified using approved cryptography.
+* Use of the biometric SHALL be limited to a single, specific device that is identified using approved cryptography.
```

Comment:
Fine, then drop 'single'.  That is misleading.  I can have multiple trusted devices using a single enrolled template on a server.